Interfaces/telegram_bot_interface.py

```diff
@@ -16,11 +16,11 @@
 
 from telegram.ext import CommandHandler, MessageHandler, Filters
 from telegram.constants import MAX_MESSAGE_LENGTH
+from telegram.utils.helpers import escape_markdown
```

Comment:
?

Reply:
It was ```from octobot_commons.pretty_printer import escape_markdown``` previously and octobot_commons.pretty_printer was importing telegram.utils.helpers, no need for this intermediary